pkg/api/persistentvolume/util.go

```diff
@@ -49,12 +56,33 @@ func GetWarningsForPersistentVolume(pv *api.PersistentVolume) []string {
 	if pv == nil {
 		return nil
 	}
-	return warningsForPersistentVolumeSpecAndMeta(nil, &pv.Spec)
+	return warningsForPersistentVolumeSpecAndMeta(nil, &pv.Spec, &pv.ObjectMeta)
+}
+
+func GetWarningsForStorageClassAnnotations(fieldPath *field.Path, meta *metav1.ObjectMeta) []string {
+	var warnings []string
+
+	if meta.Annotations != nil {
```

Comment:
no need for the nil check, a nil map returns false for lookups

Reply:
fixed